onnxruntime/core/mlas/lib/wasm_simd/SgemmKernelWasmSimd.cpp

```diff
@@ -171,11 +171,11 @@ Return Value:
         if (k > 0) {
 
             Row0AElements0 = a[0];
-            Row0AElements1 = a[1];
+            Row0AElements1 = a[0];
```

Comment:
How about line 125?

Reply:
That's is right. Because here K is 1.

Reply:
we don't need Row0AElements1. just remove this line and also the 178.